pkg/build/pipelines/ruby/require.yaml

```diff
@@ -0,0 +1,88 @@
+name: Test a Ruby package require, with optional load clause
+
+needs:
+  packages:
+    - ruby-${{inputs.version}}
+
+inputs:
+  ruby:
+    description: Which Ruby to use
+    default: DEFAULT
+  version:
+    description: Which Ruby version to use
+    default: 3.2
```

Comment:
maybe if you're letting it declare version, then don't let it declare 'ruby' ?
because what happens if i do:
```
ruby: ruby3.3
version: ruby3.2
```

I tihnk one or the other.
Either take `ruby` and just don't bother with `needs`.
or take version , set needs, and use either explicitly use `/usr/bin/ruby${{inputs.version}}` or just assume in PATH as `ruby${{inputs.version}}`.

thoughts?

Reply:
I'm kind of thinking out loud here.  I think its wierd to take both, but it was _very_ useful to in the python ones to use `DEFAULT` and pick the right /usr/bin/python3.XX when there was only one, and error in other cases.